acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature

```diff
@@ -83,12 +83,12 @@ Feature: International Phone Numbers
     Then the new user is taken to the check your phone page
     When the user clicks the support link having not received their security code
     Then the user is taken to the Contact us page in a new tab
-    When the user selects "A problem creating a GOV.UK One Login" and proceeds
+    When the user selects "A problem creating a GOV.UK One Login"
     Then the user is taken to the "A problem creating a GOV.UK One Login" page
-    When the user selects "You did not get a security code" and proceeds
+    When the user selects "You did not get a security code"
     Then the user is taken to the "You did not get a security code" page
-    When the user selects "Text message to a phone number from another country" and proceeds
-    Then the user receives confirmation that their message has been submitted
+    And the user selects "Text message to a phone number from another country"
```

Comment:
Is it worth getting rid of this `And` condition too? Does it offer us anything?

Reply:
That step is selecting the new radio button that was put in for IPN so I think there is still value in it being there as it's proving that the button exists and can be selected, even if we're no longer going to actually submit the case.

Reply:
Cool. That makes sense.